qcodes/instrument/group_parameter.py

```diff
@@ -57,9 +57,8 @@ def __init__(self,
 
         self.set = self._wrap_set(self.set_raw)
```

Comment:
shall we change ``set`` as well? :)

Reply:
Yes, I removed this line as it will be taken care of in Parameter